flipperdevices/flipperzero-firmware
# [FL-2651, FL-2863] App name in CLI loader command, RFID data edit fix

# What's new

- CLI: `loader info` command to get current running app
- RFID: fix key data edit (#1780)
- "KB" -> "KiB"

# Verification 

- Start any app, try running another one using `loader open` command
- Check `loader info` command

# Checklist (For Reviewer)
